src/definitions/modules/dropdown.js

```diff
@@ -345,11 +346,19 @@ $.fn.dropdown = function(parameters) {
             if( module.is.search() && !module.has.search() ) {
               module.verbose('Adding search input');
               $search = $('<input />')
-                .addClass(className.search)
-                .prop('autocomplete', 'off')
+                  .addClass(className.search)
+                  .prop({
+                      'autocomplete': 'off',
+                      'title': title ? title : 'search',
```

Comment:
Instead of checking if title is set just put the default value in the settings instead of the setting being `null` by default.